.travis.yml

```diff
@@ -2,13 +2,48 @@ language: android
 
 android:
   components:
-    - platform-tools
+    # Update tools and then platform-tools explicitly so lint gets an updated database. Can be removed once 3.0 is out.
```

Comment:
Might want to look at doing this if you can update to 3.0 (as a later followup)

Reply:
Nah, I'm going to punt that.